code/src/pages/dashboard/dashboard-products/DashboardProductsPage.tsx

```diff
@@ -3,15 +3,25 @@ import AddIcon from "@mui/icons-material/Add";
 import DashboardTabContainer from "@/layouts/dashboard-layout/components/dashboard-tab-container/DashboardTabContainer";
 
 import ProductsTable from "@/containers/tables/products-table/ProductsTable";
-import { mockProducts } from "@/containers/tables/products-table/ProductsTable.constants";
 
 import AppBox from "@/components/app-box/AppBox";
 import AppButton from "@/components/app-button/AppButton";
 import AppTypography from "@/components/app-typography/AppTypography";
 
 import routePaths from "@/constants/routes";
+import { useLocaleContext } from "@/context/i18n/I18nProvider";
+import { useGetManagerProductsQuery } from "@/store/api/productsApi";
 
 const DashboardProductsPage = () => {
+  const { locale } = useLocaleContext();
+  const { data, isLoading } = useGetManagerProductsQuery({ lang: locale });
+
+  if (isLoading) {
+    return <div>Loading...</div>;
+  }
```

Comment:
We also have `PageLoadingFallback` option for skeleton. I am not sure what option I should stick to. Can you help me to make a choise?

https://github.com/user-attachments/assets/10ef297a-dfb1-4bc5-bae5-ebcd6e8aee32

Reply:
> We also have `PageLoadingFallback` option for skeleton. I am not sure what option I should stick to. Can you help me to make a choise?
> 
>  loader.mp4 
> I also used such solution because it was used in orders tab. I also added a TODO for now

You left `@TODO` it's ok.
We should to investigate that